### DISCUSSION TITLE
Multiple Custom Schedules for Notifications

### DISCUSSION THREAD

**Discussion by apwestgarth** [community/community#143404] [Mobile]
Love the ability to control the schedule for push notifications.  Could be made event better by allowing different schedules for different days, rather than one time range set for all days.  Use case - I work across multiple time zones so would like to be able to set a time range for each day I want to be notified for example Monday and Friday UK business hours then Tuesday-Thursday Pacific Business Hours.  Other users may wish to configure around school or other commitments for example.
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.183.1 (145298821)
Device: iPhone 11 Pro (iOS 18.1)
</details>

**Comment by kangarruu** [comment]
Thanks for writing in @apwestgarth! I either swipe and forget, or having them clutter up my phone till I could get to them, so I personally love this idea 😄. I filed an issue with our team so we could get this feature request reviewed and potentially put on our roadmap. Keep those suggestions coming!
